### ISSUE TITLE
Create mailing list for users

### ISSUE THREAD

**Issue by radu-matei** [brigadecore/community#5]
We should have a mailing list for users.
@caniszczyk - I believe there is already a list for the maintainers - should we create another one for users, managed in the same way?

**Comment by caniszczyk** [comment]
yes you can do that, go to servicedesk.cncf.io and file an issue and we can create one for you
